docs/Materials.md.html

```diff
@@ -55,10 +55,11 @@
 # Material models
 
 Filament materials can use one of the following material models:
-- Lit (or standard)
-- Subsurface
-- Cloth
-- Unlit
+- `lit` (or standard)
```

Comment:
Instead of changing the shading model names to the programmatic versions here, you should update the `### shadingModel` section to add `specularGlossiness` to the list.

Reply:
Done